corehq/pillows/mappings/case_mapping.py

```diff
@@ -41,7 +41,8 @@
                     },
                     'type': 'nested'},
         'closed': {'type': 'boolean'},
-        'closed_by': {'type': 'string'},
+        # todo; check with team
+        'closed_by': {"type": "string", "index": "not_analyzed"},
```

Comment:
Question for reviewers.

The 3 fields changed in this mapping are best suited as keywords and ES7 explicitly needs them to be keywords for the queries we are doing against them. The concern with changing the mapping is not for ES7 to-be migrated envs, but the ones that will continue to be. Is it okay, to edit the mapping, but not trigger reindex by updating index name in those cases (since the queries do continue to work on ES2). For ES7 envs we reindex anyway, so this mapping change is not a concern.